twitchdl/commands/download.py

```diff
@@ -289,6 +298,13 @@ def _download_video(video: Video, args: DownloadOptions) -> None:
     sources = [base_uri + vod.path for vod in vods]
     targets = [cache.get_path(f"{vod.index:05d}.ts") for vod in vods]
 
+    # replace unmuted to muted for subscriptions
+    if auth_playlist:
+        sources_muted = []
+        for line in sources:
+            sources_muted.append(line.replace("-unmuted", "-muted"))
+        sources = sources_muted
+
```

Comment:
Not sure what this code does. Why replace unmuted with muted?

Reply:
if audio was removed from a video its getting empty segements / files. thats why `response.headers.get("content-length")` will fail and file merging too.
by replacing it we are getting silent audio segments

Reply:
Ah that makes sense, do you have an example video with muted segments? Can't find one at the moment. I'm writing some tests.

Reply:
luckly the link in descriptions is still online. It has 30 minutes muted audio at the beginning:

https://www.twitch.tv/videos/2411515859
twitchdl download 2411515859 --quality=audio_only

Reply:
Excellent, thanks.
It would be interesting to compare this with a public video with muted audio. I have a feeling that this substitution may be useful for public videos too. I'll keep an eye out for one.

Reply:
Did some testing, and your code is correct, only subs can hear the unmuted audio, and it must me muted when employing this workaround. I'll add a comment and maybe a log message to explain this.

BTW, the "content-length" error was a bug, because I was not checking that HTTP 200 was returned when downloading individual VODs. Fixed in 3093472e8ebc1955d0af3918ec2e81c985ebdb1d.